src/app/corporate-event/corporate-event.component.html

```diff
@@ -60,6 +60,9 @@ <h2>Please complete the form below:</h2>
       <div class="form-group">
         <textarea formControlName="additionalInfo" type="textarea" class="form-control" id="additionalInfo" placeholder="Additional Info"></textarea>
       </div>
+      <div class="form-group">
+          <re-captcha formControlName="captcha" (resolved)="resolved($event)" siteKey="6LfpwmUUAAAAAN33yLrToNNR6leqqoIoRDTu0mWH" style="border-left: 0px"></re-captcha>
```

Comment:
can you extract the `siteKey` to be configurable from a config file? Also will help for others not to have your key, no?